src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/VisitController.kt

```diff
@@ -37,20 +36,19 @@ import javax.validation.Valid
 @Validated
 @RequestMapping(name = "Visit Resource", path = ["/visits"], produces = [MediaType.APPLICATION_JSON_VALUE])
 class VisitController(
-  private val visitService: VisitService,
-  private val snsService: SnsService,
+  private val visitService: VisitService
 ) {
 
   @PreAuthorize("hasRole('VISIT_SCHEDULER')")
-  @PostMapping
+  @PostMapping("/reserve")
```

Comment:
have a look at these end points, any ideas? not sure about them, at least there self describing